test/test_overrides.py

```diff
@@ -1117,7 +1118,7 @@ def __torch_function__(self, *args, **kwargs):
                 # Function that handles torch_function in C++
                 torch.abs(a)
 
-@skipIfCrossRef
+@unittest.skipIf(TEST_WITH_CROSSREF, "not run with crossref")
```

Comment:
How is this different from [skipIfCrossRef](https://github.com/pytorch/pytorch/blob/d0baf5792a01cc5554d884b8579cc82239b87510/torch/testing/_internal/common_utils.py#L813)?

Reply:
Apparently skipIfCrossRef doesn't work when applied to an entire test class